packages/edit-site/src/components/dataviews/style.scss

```diff
@@ -38,6 +38,19 @@
 	tr {
 		border-bottom: 1px solid $gray-100;
 	}
+	.block-editor-block-preview__container {
```

Comment:
This style seem misplaced to me, the data views is generic and this is for a specific component.

Reply:
You're totally right! I'll move them.